src/components/Hub/HubOverviewActionsContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import HubOverviewAction from "./HubOverviewAction";
+
+const HubOverviewActionsContainer = () => {
+  const history = useHistory();
+
+  const hubOverviewActionsData = [
```

Comment:
Also here would make sense to introduce a type

Reply:
Yeah, I'll make this change